R/gtrends.R

```diff
@@ -289,17 +341,20 @@ plot.gtrends <- function(x,
                   resolution = resolution)
   
   if (type == "trend") {
-    #z <- as.zoo.gtrends(x)
     
     df <- x$trend
-    df <- tidyr::gather_(df, "keyword", "hit", names(df)[3:ncol(df)])
+    df <- tidyr::gather_(df, "keyword", "hit", na.omit(names(df)[3:ncol(df)]), 
+                         convert = TRUE)
+    
+    df$start <- as.POSIXct(df$start)
     
-    p <- ggplot(df, aes_string(x = "end", y = "hit", color = "keyword")) +
+    p <- ggplot(df, aes_string(x = "start", y = "hit", color = "keyword")) +
       geom_line() +
       xlab("Date") +
       ylab("Search hits") +
       ggtitle("Interest over time") +
-      theme_bw()
+      theme_bw() +
+      scale_x_datetime(labels = scales::date_format("%Y/%m/%d"))
```

Comment:
You seem to have been infected by the Hadleyverse bug where for every problem a Hadley package must be used.

Is this not the as 

```r
R> format(Sys.Date(), "%Y/%m/%d")
[1] "2015/11/24"
R> 
```

And for that matter ISO date `%Y-%m-%d` may be more standard.